algorithms/bellman_ford.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import graphviz
+
+## Defining classes for vertex and Graph
+
+class vertex :
+    def __init__(self,distance=0,pre=0):
+        self.d=distance
+        self.p=pre
+
+        
+class Graph :
+    def __init__(self,vertices=0,edges=0):
+
+        self.vertices=vertices
+        self.edges=edges
+   
+    def single_source(self,j):
+        k=list(self.edges.values())
+        k=np.array(k)
+        k=abs(k)
+        max_edge= k.sum()
+        for i in self.vertices:
+            self.vertices[i].d= max_edge+1
+        self.vertices[j].d=0
+        return None
+            
+    def relax(self,j):
+        if self.vertices[int(j[3])].d > self.vertices[int(j[1])].d +self.edges[j]:
+           self.vertices[int(j[3])].d =  self.vertices[int(j[1])].d +self.edges[j]
+           self.vertices[int(j[3])].p= int(j[1])
+           return None
+             
+        
+
+
+    def bellman_ford(self,j):
+        self.single_source(j)
+        for i in range(len(self.vertices)-1):
+            for i in self.edges:
+                self.relax(i)
+        for i in self.edges:
+            if self.vertices[int(i[3])].d >  self.vertices[int(i[1])].d +self.edges[i]:
+                return False
+        return self.vertices
+
+
+
```

Comment:
Remove unnecessary empty lines.

Reply:
I have committed to all the changes suggested by you, except this one change of removing empty lines. Can this be done on GitHub?